Petulia/Petfinder-Info.plist

```diff
@@ -3,8 +3,8 @@
 <plist version="1.0">
 <dict>
 	<key>API_KEY</key>
-	<string>_ADD_YOUR_API_KEY_HERE</string>
+	<string>PnorMn2oi77FdbR6DAqCkRo34DNiE9UUYGNazLecA2XuWi7ZtW</string>
```

Comment:
Are you pushing this file as well? You shouldn't be pushing your API Key. Check if the gitignore is working the right way.